templates/home.html

```diff
@@ -22,58 +22,70 @@ <h2>Ukrainianconflict-live</h2>
 {% block pagetitle %}Dashboard{% endblock %}
 
 {% block content %}
-    <div class="row">
+    <div class="row  map-section">
+        
         <h3>Maps</h3>
+        <div class="toggle-parent" data-show="1">
+            <div class="toggle-element">
+                <button data-val="1">1</button>
```

Comment:
Personally, I'll go for labels instead of indexes, maybe with a select instead of buttons?